### DISCUSSION TITLE
how to use browser option and auto add to cart? (noob)

### DISCUSSION THREAD

**Discussion by Opt147** [jef/streetmerchant#1570] [Help]
I wanna know how to use the browser option sort of like on this guys stream (top right) https://www.youtube.com/watch?v=o3UlT5UDRC8 and also the option to add to cart. please don't judge I tried the whole day to get this going but my lack of coding experience holds me back. I just want a 3080 thanks :)

**Comment by MastermindFIL** [comment]
Go into the .env file and go all the way to the top and there should be a thing called "auto_add_to_cart" and just type true next to it. I'm not sure if you have to add quatations but you can test it out. But I need help and for some reason the pages aren't refreshing. I used this before but now I re did it cause I accidentally broke it. Like when you type "npm run start" it shows everything but it dosnt show when it says for example "5600x" is out of stock.

**Reply by Opt147** [reply]
have you tried deleting all of the files and redownloading it? if its not that it might be the current version but im not sure.

**Comment by jef** [comment]
```dotenv
AUTO_ADD_TO_CART=true
OPEN_BROWSER=true
```

Is what this should look like. You also don't have to set this values as they are `true` by default.

**Reply by Opt147** [reply]
I see chromium running in the background of task manager but chromium it self never opens on my desktop.

**Reply by jef** [reply]
`HEADLESS=false` is what you're looking for then!
